masq_lib/src/test_utils/utils.rs

```diff
@@ -3,8 +3,11 @@
 use std::fs;
 use std::path::PathBuf;
 
-pub const DEFAULT_CHAIN_ID: u8 = 3u8; //For testing only
-pub const TEST_DEFAULT_CHAIN_NAME: &str = "ropsten"; //For testing only
+pub const TEST_DEFAULT_CHAIN_ID: u8 = 2;
+pub const TEST_DEFAULT_CHAIN_NAME: &str = "ropsten";
```

Comment:
I guess I'm unclear why `DEFAULT_CHAIN_NAME` and `DEFAULT_CHAIN_ID` within the `test_utils` namespace needs to be declared with `TEST_` as a prefix. Is it too confusing? It does make one check to make sure the correct module was imported?